core/blockly.js

```diff
@@ -132,6 +132,12 @@ Blockly.cache3dSupported_ = null;
  */
 Blockly.theme_ = null;
 
+/**
+ * Blockly core version.
+ * @define {string}
+ */
+Blockly.version = "unset";
```

Comment:
I think this should be a constant.  Thus `Blockly.VERSION`
Also, Google prefers single quoted strings unless there's a reason otherwise (e.g. JSON, or strings with single quotes)
https://google.github.io/styleguide/javascriptguide.xml?showone=Strings#Strings

Reply:
Cool, thanks for the tip.